PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -11,9 +11,8 @@
 
 #include "InputSystem.hpp"
 
-namespace Poly 
+namespace Poly
 {
-	class World;
```

Comment:
Oooh, good catch. It works, because it's forward-declared in InputSystem.hpp, which is included here. It's weird that I removed it though, must have been a mistake.